bpf/bpf_host.c

```diff
@@ -101,18 +101,18 @@ ipcache_lookup_srcid6(struct __ctx_buff *ctx)
 	struct remote_endpoint_info *info = NULL;
```

Comment:
`also update all update variables` -- too many `update`s

Reply:
Fixed in newest commit